keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -193,9 +222,20 @@ Success will be determined by stability of the provided mechanisms and ease of u
 
 - 05/18/2018: initial design
 - 06/13/2018: updated design
+- 07/30/2018: dynamic policy addition
 
 ## Alternatives
 
 ### Generalized Dynamic Configuration
 
 We could strive for all kube-apiserver flags to be able to be dynamically provisioned in a common way. This is likely a large task and out of the scope of the intentions of this feature.
+
+### Policy Override
+
+There has been discussion over whether the policy configured by api server flags should limit the policies configured dynamically. This would allow a cluster admin to narrowly define what is allowed to be logged by the dynamic configurations. While this has upsides it was ruled out for the following reasons: 
+
+* It would limit user story #4 in the ability to quickly turn up logging when needed 
+* It could prove difficult to understand as the policies themselves are fairly complex 
+* The use of CRDs would be difficult to bound
+
+The dynamic policy feature is gated by runtime flags. This still provides the cluster provisioner a means to limit audit logging to the single runtime object if needed.
```

Comment:
feature gates aren't intended as long-term configuration options. If this is a desirable knob, we should add a non-feature gate flag for it.